README.md

```diff
@@ -13,12 +15,17 @@ Download the executable from the [Releases](../../releases) page of this reposit
 
 - Windows: `bts.exe`
   - Run once with administrator privileges to enable integration with Windows "Open with" menu for .xlsx files
-  - After that, the program will appear as an option when right-clicking Excel files
+  - After that, the program will appear as an option when right-clicking Excel files. Click "Choose another app", then "bts.exe", and then choose "Just Once" to avoid having bloom-translation-spreadsheet become your default app for "opening" .xlsx files.
```

Comment:
I keep accidentally getting bts set as my default program for opening spreadsheets, and then when I double click on spreadsheets the terminal flashes open very briefly and sometimes I don't even notice it and then nothing else happens. I'm worried this will be confusing if others have it happen to them, could see if there's a programmatic way to prevent it